.github/DISCUSSION_TEMPLATE/RFC.yml

```diff
@@ -0,0 +1,82 @@
+title: "Feature request"
```

Comment:
Maybe change it to "RFC" instead of "Feature Request" so it's specifically used by us only for that purpose?